### ISSUE TITLE
MVVM Isuue

### ISSUE THREAD

**Issue by evyasafhouzz** [sergdort/CleanArchitectureRxSwift#76]
`PostsViewModel` is exposing to the view the Post mode as output: `let selectedPost: Driver<Post>`

My question is:
`PostsViewController` knows `Post` data model which means that the view knows the model.
As far as I understand MVVM architecture the UI (view) doesn't need to know the model
(Post). 

What do you think?  is there a place to change the `selectedPost` parameter to be:
`let selectedPost: Driver<PostViewModel>` ?

**Comment by sergdort** [comment]
Yeah definitely, I am just being lazy here 🙈because there is no formatting involved before rendering information in a view.

In this case it would be literally one to one mappings of properties 🤷‍♂️
